web/src/components/apps/DashboardGraphsCard.jsx

```diff
@@ -227,38 +282,47 @@ export default class DashboardGraphsCard extends React.Component {
   }
 
   render() {
-    if (this.props.isHelmManaged === true){
-      return(
-        <div></div>
-      )
+    if (this.props.isHelmManaged === true) {
+      return <div></div>;
     }
     const { prometheusAddress, metrics } = this.props;
-    const { promValue, showConfigureGraphs, savingPromError, savingPromValue } = this.state;
+    const { promValue, showConfigureGraphs, savingPromError, savingPromValue } =
+      this.state;
 
     return (
-      <div className={`${!prometheusAddress ? "inverse-card" : ""} dashboard-card flex-column flex1`}>
+      <div
+        className={`${
+          !prometheusAddress ? "inverse-card" : ""
+        } dashboard-card flex-column flex1`}
+      >
         <div className="flex justifyContent--spaceBetween alignItems--center">
-          <p className="u-fontSize--large u-textColor--primary u-fontWeight--bold">Monitoring</p>
+          <p className="u-fontSize--large u-textColor--primary u-fontWeight--bold">
+            Monitoring
+          </p>
           <div className="flex alignItems--center">
             <span className="icon clickable dashboard-card-settings-icon u-marginRight--5" />
-            <span className="replicated-link u-fontSize--small" onClick={this.toggleConfigureGraphs}>Configure Prometheus Address</span>
+            <span
+              className="replicated-link u-fontSize--small"
+              onClick={this.toggleConfigureGraphs}
+            >
+              Configure Prometheus Address
+            </span>
           </div>
         </div>
-        {prometheusAddress ?
-          <div className="Graphs-wrapper">
-            {metrics.map(this.renderGraph)}
-          </div>
-          :
+        {prometheusAddress ? (
+          <div className="Graphs-wrapper">{metrics.map(this.renderGraph)}</div>
+        ) : (
           <div className="flex flex1 justifyContent--center u-paddingTop--50 u-paddingBottom--50 u-position--relative">
             <ConfigureGraphs
               updatePromValue={this.updatePromValue}
               promValue={promValue}
               savingPromValue={savingPromValue}
               savingPromError={savingPromError}
               onPromValueChange={this.onPromValueChange}
+              placeholder={prometheusAddress}
```

Comment:
The files in this PR were linted to make them prettier. This line and line 341 are the main changes in this file (providing a placeholder for the Prometheus address).